feat(changelog): containers-added-deploy-containers-from-public 2023-02-07 (#1277)
scaleway/docs-content: changelog/february2023/2023-02-07-containers-added-deploy-containers-from-public.mdx

---

> title: Deploy Containers from public registries

Reviewer: ```suggestion
title: Deploying Containers from public registries is now possible!
```

status: added
author:
    fullname: 'Join the #[serverless-containers] channel on Slack.'
    url: 'https://scaleway-community.slack.com/archives/C02J1M100AJ'
date: 2023-02-07
category: serverless
product: containers
---

You can now deploy container images stored in public registries such as Docker Hub or your personal container registry to Serverless Containers. 

> Simply write the image adresse (for example `wordpress:latest` do deploy the latest Wordpress image from [Docker Hub](https://hub.docker.com/) ) in the Console, Scaleway's API, Terraform or Serverless Framework 

Reviewer 2: ```suggestion
Simply write the image address (for example: `wordpress:latest` do deploy the latest WordPress image from [Docker Hub](https://hub.docker.com/) ) in the Console, Scaleway's API, Terraform or Serverless Framework 
```

> Simply write the image adresse (for example `wordpress:latest` do deploy the latest Wordpress image from [Docker Hub](https://hub.docker.com/) ) in the Console, Scaleway's API, Terraform or Serverless Framework 

Reviewer: ```suggestion
Simply write the image address (for example `wordpress:latest` to deploy the latest Wordpress image from [Docker Hub](https://hub.docker.com/) ) in the [Scaleway console](https://console.scaleway.com), Scaleway's API, Terraform or Serverless Framework 
```
